index.js

```diff
@@ -0,0 +1,3 @@
+const server = require("./server");
+
+server.listen(4000, () => {  })
```

Comment:
you'll want a console.log in the callback to give yourself indication when its running